apps/frontend/src/utils/math.ts

```diff
@@ -0,0 +1,50 @@
+import { isBigNumberish } from '@ethersproject/bignumber/lib/bignumber';
+
+import { BigNumberish, BigNumber } from 'ethers';
+import { formatUnits, parseUnits } from 'ethers/lib/utils';
+
+const DEFAULT_UNIT = 18;
+const DEFAULT_DECIMALS = 6;
+
+const unitNames = ['wei', 'kwei', 'mwei', 'gwei', 'szabo', 'finney', 'ether'];
+
+// helper function to convert any type of ethers value to wei.
+export const toWei = (
+  value: BigNumberish,
+  unitName: BigNumberish = DEFAULT_UNIT,
+): BigNumber => {
+  if (isBigNumberish(value)) {
+    if (typeof unitName === 'string') {
+      const index = unitNames.indexOf(unitName);
+      if (index !== -1) {
+        unitName = 3 * index;
+      }
```

Comment:
```suggestion
unitName = units[unitName];
```
I think this is a better approach compared to multiplying an index from the `unitNames` array with a seemingly random number (`3`).